rca/home/models.py

```diff
@@ -1,29 +1,183 @@
+        related_name="+",
+    )
+    video = models.URLField(blank=True)
+    video_caption = models.CharField(
+        blank=True,
+        max_length=80,
+        help_text="The text dipsplayed next to the video play button",
+    )
+    sub_heading = models.CharField(
+        max_length=125, blank=True, help_text="The title below the image"
+    )
+    page_title = models.CharField(
+        max_length=125, blank=True, help_text="A title for the linked related page"
+    )
+    page_summary = models.CharField(
+        max_length=250, blank=True, help_text="A summary for the linked related page"
+    )
+    page_link_url = models.URLField(blank=True, help_text="A url to a related page")
+
+    panels = [
+        FieldPanel("heading"),
+        ImageChooserPanel("image"),
+        FieldPanel("video"),
+        FieldPanel("video_caption"),
+        FieldPanel("sub_heading"),
+        FieldPanel("page_title"),
+        FieldPanel("page_summary"),
+        FieldPanel("page_link_url"),
+    ]
+
+    def __str__(self):
+        return self.heading
+
+    def clean(self):
+        errors = defaultdict(list)
+        if self.video and not self.video_caption:
+            errors["video_caption"].append("Please add a caption for the video")
+
+        if errors:
+            raise ValidationError(errors)
+
+
+class HomePagePartnershipBlock(models.Model):
+    # Partnership module
+    source_page = ParentalKey("HomePage", related_name="partnerships_block")
+    title = models.CharField(max_length=125)
+    summary = models.CharField(max_length=250)
+    slides = StreamField([("slide", SlideBlock())])
+    panels = [FieldPanel("title"), FieldPanel("summary"), StreamFieldPanel("slides")]
+
+    def __str__(self):
+        return self.title
+
+
+class HomePageStatsBlock(models.Model):
+    source_page = ParentalKey("HomePage", related_name="stats_block")
+    title = models.CharField(max_length=125)
+    statistics = StreamField([("statistic", StatisticBlock())])
+    background_image = models.ForeignKey(
+        get_image_model_string(),
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    panels = [
+        FieldPanel("title"),
+        ImageChooserPanel("background_image"),
+        StreamFieldPanel("statistics"),
+    ]
+
+    def __str__(self):
+        return self.title
+
+
 class HomePage(BasePage):
     template = "patterns/pages/home/home_page.html"
 
     # Only allow creating HomePages at the root level
     parent_page_types = ["wagtailcore.Page"]
 
-    strapline = models.CharField(blank=True, max_length=255)
-    call_to_action = models.ForeignKey(
-        "utils.CallToActionSnippet",
-        blank=True,
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
         null=True,
+        blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    hero_colour_option = models.PositiveIntegerField(
```

Comment:
Why not `PositiveSmallIntegerField`?
https://docs.djangoproject.com/en/2.2/ref/models/fields/#positivesmallintegerfield